components/yaml/introduction.rst

```diff
@@ -148,6 +148,16 @@ error if something goes wrong by adding the filename to the message.
 
 .. _components-yaml-dump:
 
+Objects for Mappings
+....................
+
+Yaml mappings_ are basically associative arrays. You can instruct the parser to return
```

Comment:
For interal links, we can use the `ref` role:

```rst
Yaml :ref:`mappings <yaml-format-collections>` are [...]
```

And we then need the label in the `/components/yaml/yaml_format.rst` file:

```rst
.. _yaml-format-collections:

Collections
-----------
```